app.js

```diff
@@ -62,29 +29,69 @@ var getCookiesPerHour = function(objectName){
   }
 };
 
+CookieStore.prototype.makeTheTable = function(tableToDrawInto){
+  var hourlySalesRowEl = document.createElement('tr');
+  console.log(hourlySalesRowEl);
```

Comment:
You're starting to get a bajillion console log statements, and after a while it starts to get difficult to figure out what is what. You can make that a lot clearer by adding a second argument to the console log to label what you're showing, like this: 

```javascript
console.log('hourlySalesRowEl: ', hourlySalesRowEl);
```